### PR TITLE
Only pass down single line trigger comment

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- When a multiline PR trigger is used,
  only pass down the line that fired
  the job. The rest is unnecessary.
- Build.groovy uses the comment but is
  expecting it to be a single line.
- There was a bug where if you had the keyword
  'depends' in your comment then the code would
  go down the path of looking for dependent changes.

Follows #7509
[skip ci]

Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
Depends on nothing
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/2267/
https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/2268/

**Comment by AdamBrousseau** [comment]
Example of the failure
https://ci.eclipse.org/openj9/job/Build_JDK8_ppc64le_linux_Personal/592/console
```
ghprbCommentBody = Depends #8098 \r\n\r\nJenkins compile plinux jdk8,jdk11
```
```
09:43:29  Look for Dependent Changes
[Pipeline] echo
09:43:29  PullRequest source repo appears to be OpenJ9
[Pipeline] error
[Pipeline] }
[Pipeline] // stage
[Pipeline] cleanWs
09:43:29  [WS-CLEANUP] Deleting project workspace...
09:43:29  [WS-CLEANUP] Deferred wipeout is disabled by the job configuration...
09:43:32  [WS-CLEANUP] done
[Pipeline] }
[Pipeline] // timeout
[Pipeline] }
[Pipeline] // node
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // timestamps
[Pipeline] }
[Pipeline] // timeout
[Pipeline] End of Pipeline
ERROR: Bad Depends Repo: '' in '#8098'
Finished: FAILURE
```

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
Depends on nothing
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
@vsebe for review

File: buildenv/jenkins/common/pipeline-functions.groovy [hunk 1]
```diff
@@ -677,7 +677,7 @@ def setup_pull_request() {
        Jenkins test extended <platform>
        It was tested*/
 
-    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.split(/\\r?\\n/)
+    def PARSED_BY_NEWLINE_COMMENT = ghprbCommentBody.split(/\\r?\\n/)
     for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
```
Comment (vsebe) [inline buildenv/jenkins/common/pipeline-functions.groovy]: `PARSED_BY_NEWLINE_COMMENT` and `COMMENT`  are local variables. Please use lower case notation and rename COMMENT (e.g. line) as there is also a local variable `comment`.

**Comment by vsebe** [comment]
Please fix typo in the commit message: ` dependnet `

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
Jenkins test sanity zlinux jdk11

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk11

File: buildenv/jenkins/common/pipeline-functions.groovy (hunk 1 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/common/pipeline-functions.groovy]: Updated the 2 functions I touched. Launching compile and test builds to prove those changes.

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
@vsebe I think this is ready

File: buildenv/jenkins/common/pipeline-functions.groovy [hunk 2]
```diff
@@ -677,18 +677,18 @@ def setup_pull_request() {
...
 
-    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.split(/\\r?\\n/)
-    for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
-        def comment = COMMENT.toLowerCase().tokenize(' ')
-        if (("${comment[0]}" == "jenkins") && (("${comment[1]}" == "compile") || ("${comment[1]}" == "test"))) {
-            setup_pull_request_single_comment(comment)
+    def parsedByNewlineComment = ghprbCommentBody.split(/\\r?\\n/)
+    for (comment in parsedByNewlineComment) {
+        def lowerCaseComment = comment.toLowerCase().tokenize(' ')
+        if (("${lowerCaseComment[0]}" == "jenkins") && (("${lowerCaseComment[1]}" == "compile") || ("${lowerCaseComment[1]}" == "test"))) {
```
Comment (vsebe) [inline buildenv/jenkins/common/pipeline-functions.groovy]: maybe simplify 
`(("${lowerCaseComment[1]}" == "compile") || ("${lowerCaseComment[1]}" == "test"))`
to 
`(["compile", "test"].contains(lowerCaseComment[1])`

Comment (AdamBrousseau) [reply buildenv/jenkins/common/pipeline-functions.groovy]: changed

**Comment by AdamBrousseau** [comment]
Depends on nothing
Jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
@vsebe 
Still seems to be working.

**Comment by pshipton** [comment]
There is a merge conflict.

**Comment by AdamBrousseau** [comment]
conflict resolved.

**Pull request merged**